ocaml/xapi/xapi.ml

```diff
@@ -632,7 +632,12 @@ let set_stunnel_legacy_inv ~__context () =
     );
   Stunnel.set_legacy_ciphersuites !Xapi_globs.ciphersuites_legacy_outbound;
   let s = Xapi_inventory.lookup Xapi_inventory._stunnel_legacy ~default:"true" in
-  let legacy = try
+  let cc_mode = try (
```

Comment:
This could be done in a `stunnel_should_use_legacy` function avoiding duplication of code
(the cc mode read and stunnel legacy inventory lookup)

Reply:
Yes, will update. Thanks.